server/engine/src/main/java/org/eclipse/lsp/cobol/core/engine/dialects/idms/IdmsCopybookDescriptor.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2022 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.engine.dialects.idms;
+
+import lombok.Data;
+import org.antlr.v4.runtime.Token;
+import org.eclipse.lsp.cobol.core.IdmsCopyParser;
+import org.eclipse.lsp.cobol.core.IdmsParser;
+import org.eclipse.lsp.cobol.core.engine.dialects.DialectUtils;
+import org.eclipse.lsp.cobol.core.model.Locality;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.Range;
+
+/*
+ * copyIdmsOptions
+ *     : (RECORD copyIdmsSource versionClause? (REDEFINES cobolWord)?)
+ *     | (FILE copyIdmsSource versionClause?)
+ *     | ((MAP | MAP_CONTROL) copyIdmsSource)
+ *     | (MODULE? copyIdmsSource versionClause?)
+ *     ;
+ *
+ * copyIdmsSource
+ *     : copySource
+ *     ;
+ *
+ * copySource
+ *    : (literal | cobolWord | SUBSCHEMA_NAMES) ((OF | IN) copyLibrary)?
+ *    ;
+ *
+ * copyLibrary
+ *    : literal | cobolWord
+ *    ;
+ */
+
+/**
+ * Description of IDMS copybook
+ */
+@Data
+public class IdmsCopybookDescriptor {
+  String name;
```

Comment:
please don't break encapsulation rules here, make those fields private

Reply:
They are, it's Lombok magic

Reply:
I doubt that, the thing that lombok does with `@Data` is creating getters and setters (and something more), but it's possible to threat that as dumb structure with `descriptor.name =`